### PR TITLE
Added 64bit libraries for Android

### PR DESCRIPTION
**Pull request by Waldegger**
added necessary libs in extlibs\libs-android\arm64-v8a and extlibs\libs-android\x86_64 to create 64 bit Programs for those ABIs on Android

### CODE REVIEW

**Comment by eXpl0it3r** [comment]
Can you please target `2.6.x` instead of `master`?

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Comment by ChrisThrasher** [comment]
Does this not also need some CMake tie-in to work? How does the 64-bit Android build know where to find these libraries?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,6 +32,10 @@ jobs:
           config: { name: x86, flags: -DCMAKE_ANDROID_ARCH_ABI=x86 -DCMAKE_SYSTEM_NAME=Android -DSFML_BUILD_TEST_SUITE=FALSE -DCMAKE_ANDROID_NDK=$GITHUB_WORKSPACE/android-ndk-r18b -DCMAKE_ANDROID_NDK_TOOLCHAIN_VERSION=clang -DCMAKE_ANDROID_STL_TYPE=c++_shared -DCMAKE_ANDROID_API=26 }
         - platform: { name: Android, os: ubuntu-latest }
           config: { name: armeabi-v7a, flags: -DCMAKE_ANDROID_ARCH_ABI=armeabi-v7a -DCMAKE_SYSTEM_NAME=Android -DSFML_BUILD_TEST_SUITE=FALSE -DCMAKE_ANDROID_NDK=$GITHUB_WORKSPACE/android-ndk-r18b -DCMAKE_ANDROID_NDK_TOOLCHAIN_VERSION=clang -DCMAKE_ANDROID_STL_TYPE=c++_shared -DCMAKE_ANDROID_API=26 }
+        - platform: { name: Android, os: ubuntu-latest  }
+          config: { name: arm64-v8a, flags: -DCMAKE_ANDROID_ARCH_ABI=arm64-v8a -DCMAKE_SYSTEM_NAME=Android -DSFML_BUILD_TEST_SUITE=FALSE -DCMAKE_ANDROID_NDK=$GITHUB_WORKSPACE/android-ndk-r18b -DCMAKE_TOOLCHAIN_FILE=$GITHUB_WORKSPACE/android-ndk-r18b/build/cmake/android.toolchain.cmake -DCMAKE_ANDROID_NDK_TOOLCHAIN_VERSION=clang -DCMAKE_ANDROID_STL_TYPE=c++_static -DANDROID_PLATFORM=android-21 }
```
[the remark this answers is not in the document]
Comment (Waldegger) [reply .github/workflows/ci.yml:36]: I gave the options with which I have compiled the libraries. I did not know of the existence of this file to take inspiration. Most probably the other options would work as well. Shall I try and change them?

**Comment by kimci86** [comment]
> Does this not also need some CMake tie-in to work? How does the 64-bit Android build know where to find these libraries?

The variable `CMAKE_ANDROID_ARCH_ABI` is already used in `src/SFML/CMakeLists.txt` to use the right library folder.

File: .github/workflows/ci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kimci86) [reply .github/workflows/ci.yml:36]: Yes, I think you should try so it is clear that there are no configuration magic tricks specific to each architecture.

[the remark this answers is not in the document]
Comment (Waldegger) [reply .github/workflows/ci.yml]: At least on my machine I cannot build succesfully without setting -DCMAKE_TOOLCHAIN_FILE

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Comment by eXpl0it3r** [comment]
Thanks a lot for this very valuable contribution! 🥳

**Pull request merged**